pom.xml

```diff
@@ -1557,6 +1557,9 @@ under the License.
 
 						<!-- AWS SDK config that does not support license headers -->
 						<exclude>**/awssdk/global/handlers/execution.interceptors</exclude>
+
+						<!-- Splunk config for MiniClusterEntryPoint -->
+						<exclude>flink-container/src/main/java/org/apache/flink/container/entrypoint/MiniClusterEntryPoint.java</exclude>
```

Comment:
what is this exclusion for? LGTM otherwise

Reply:
the compilation fails because there is no Apache License present MiniClusterEntryPoint file. Here we are using Splunk License for MiniClusterEntryPoint. Adding that exclusion prevents the check during compilation